tests/test_utils.py

```diff
@@ -0,0 +1,14 @@
+from cleo._utils import format_time
+import pytest
+
+
+@pytest.mark.parametrize("input_secs,expected", [
+    (0.1, "< 1 sec"),
+    (1.0, "1 sec"),
+    (2.0, "2 secs"),
+    (59.0, "59 secs"),
+    (60.0, "1 min"),
+    (120.0, "2 mins"),
```

Comment:
@branchvincent do these test cases match your expectations? (this is failing, currently)

Reply:
yup, makes sense to me

Reply:
Cool. That means there's a bug with the current implementation.
I'll need to find some time to revisit this. Was a while ago...

Reply:
haha of course! take your time, thanks a lot for helping out